docs/source/metabase.py

```diff
@@ -132,6 +133,10 @@ def finish(self) -> None:
             self._put(f'table/{table["id"]}', json={
                 'description': description, 'points_of_interest': points_of_interest})
 
+        # Quit early if no table has been found
```

Comment:
but why are no tables found? the v2 tables still exist, don't they?

Reply:
They’re not in the Sphinx documentation anymore, that’s why they’re not available here.

Reply:
Oh… I guess this makes sense. And the existing documentation is already in Metabase so it's fine to not update it.